storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.txt

```diff
@@ -77,32 +77,41 @@ In order for snapshot to be consistent you need to prepare the data:
 .. code-block:: console
 
```

Comment:
Update the example to remove TheMaster and TheSlave.

Reply:
This is the old description of replication.  As you can see it as the txt extension which effectively excludes it from the rendered result: this section won't appear in either html or pdf output. It stays here just to have a convenient reference to be able to verify that essential information is not lost.